### PR TITLE
Add support of callback function after add element or after remove

### PR DESCRIPTION
**Pull request by Gulivertx**
Hello,
For my need I added ability to add a callback function after adding element or deleting element.
Config looks like this :

```javascript
a2lix_lib.sfCollection.init({
            collectionsSelector: this.selector,
            manageRemoveEntry: true,
            lang: {
                add: 'Ajouter une nouvelle date',
                remove: 'Supprimer'
            },
            afterAdd: (collection, element) => {
                console.log(collection, element)
            },
            afterRemove: (collection, element) => {
                console.log(collection, element)
            }
        });
```

Should you merge into your master branch ?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```
Comment (garak) [inline .gitignore]: this is wrong. You should keep such value in a global ignore file. See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@a2lix/symfony-collection",
-  "version": "0.4.0",
+  "version": "0.4.1",
```
Comment (garak) [inline package.json]: This is not a bugfix, it's a new feature. It should go in a minor version, not in a patch version. So, 0.5.0.

File: .gitignore (hunk 1 above)
Comment (Gulivertx) [reply .gitignore]: Done

File: package.json (hunk 2 above)
Comment (Gulivertx) [reply package.json]: Done

**Comment by webda2l** [comment]
Hello, I close, the entry.add.onBeforeFn & entry.remove.onAfterFn of the 0.5.1 offer enough flexibility to handle this case IMO. See https://github.com/a2lix/symfony-collection#how-to

**Pull request closed without merging**